src/common/paddle.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  type Environment,
+  Paddle,
+  type Subscription,
+} from '@paddle/paddle-node-sdk';
+import { logger } from '../logger';
+
+export const paddleInstance = new Paddle(process.env.PADDLE_API_KEY, {
+  environment: process.env.PADDLE_ENVIRONMENT as Environment,
+});
+
+type CancelSubscriptionProps = {
+  subscriptionId: string;
+};
+export const cancelSubscription = ({
+  subscriptionId,
+}: CancelSubscriptionProps): Promise<Subscription> | undefined => {
+  logger.info(
+    {
+      type: 'paddle',
+      subscriptionId,
+    },
+    'Subscription cancelled user deletion',
```

Comment:
It's not really true, we could use this util for other cases, so ideally this log is in the directive where we call `cancelSubscription` on user deletion

Reply:
Good point :)